docs/book/user-guide/best-practices/quick-wins.md

```diff
@@ -321,7 +341,163 @@ zenml artifact-store register my_store \
 
 Learn more: [Secret Management](https://docs.zenml.io/getting-started/deploying-zenml/secret-management), [Working with Secrets](https://docs.zenml.io/how-to/project-setup-and-management/interact-with-secrets)
 
-## 8 Hook your Git repo to every run
+## 8 Run smoke tests locally before going to the cloud
+
+**Why** -- significantly reduce iteration and debugging time by testing your pipelines with a local Docker orchestrator before deploying to remote cloud infrastructure. This approach gives you fast feedback cycles for containerized execution without waiting for cloud provisioning, job scheduling, and data transfer—ideal for development, troubleshooting, and quick feature validation.
+
+```bash
+# Install the local Docker orchestrator (if not already installed)
+zenml integration install docker -y
```

Comment:
wrong, that isnt a thing. docker comes with zenml. however you need to make sure docker is installed and active on your machine (you can check with doing `docker ps`  and it should give some results